assets/scss/components/_login-modal.scss

```diff
@@ -0,0 +1,79 @@
+.login-modal {
+	width: 426px;
+	height: 465px;
```

Comment:
I wouldn't set a default height here.  Let the content set the height.  The width is fine though.